backfill.md

```diff
@@ -0,0 +1,83 @@
+# How to Backfill history_transactions table using tx_meta (xdr)
+
+In cases of new column requests/addressing bugs, airflow backfill jobs are needed to be kicked off. This process is both time and money consuming.
+
+This document outlies methods to extract required field from transaction envelope (tx_meta). This documents takes an example of `fee_account_muxed` field. However, it can be extended to other fields as well.
+
+We will use [js-stellar-base](https://github.com/stellar/js-stellar-base) library to parse the XDRs and then use Javascript UDF in Bigquery to apply the transformation to the dataset.
+
+Referred medium article: [Using NPM Library in Google BigQuery UDF](https://medium.com/analytics-vidhya/using-npm-library-in-google-bigquery-udf-8aef01b868f4)
+
+# Setting up JS UDF in Bigquery
+
+```
+git clone https://github.com/stellar/js-stellar-base.git
+cd js-stellar-base
+yarn
+npx webpack --config config/webpack.config.browser.js
+```
+
+Above will create following files in `js-stellar-base/dist/` directory:
+
+- stellar-base.js
+- stellar-base.min.js
+
+Copy above files in a GCS bucket.
+
+# Writing JS Function
+
+Following is example to extract FeeAccountMuxed:
+
+```
```

Comment:
suggestion: add `javascript` tag here for syntax highlighting

Reply:
Addressed in https://github.com/stellar/stellar-etl/pull/273/commits/d5884a2f89388cb43db21a46ececa753eaee3a9e